tests/apollo/util/bft.py

```diff
@@ -223,11 +236,14 @@ async def test_with_bft_network():
                                         bft_network.test_start_time = time.time()
                                         await async_fn(*args, **kwargs, bft_network=bft_network)
                             await test_with_bft_network()
+                    ApolloTest.FAILED_CASES.pop(subtest_id)
+                    test_instance.register_errors()
+
         return wrapper
     return decorator
 
 MAX_MSG_SIZE = 64*1024 # 64k
-REQ_TIMEOUT_MILLI = 5000
+REQ_TIMEOUT_MILLI = 5000 * 2
```

Comment:
is it doubled in purpose?
It means that in all the tests that are waiting for a client request to timeout, it will take ~10 seconds

Reply:
Yes, this was a change proposed by you because of the 5 seconds we wait for replicas with UDP connections

Reply:
If its only for UDP setups, maybe it is worth doubling the timeout only for those cases.

Reply:
Makes it more difficult to debug timeouts that way